src/PointwiseFunctions/Hydro/EquationsOfState/Enthalpy.cpp

```diff
@@ -0,0 +1,451 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "PointwiseFunctions/Hydro/EquationsOfState/Enthalpy.hpp"
+
+#include <cmath>
+#include <functional>
+#include <numeric>
```

Comment:
add: `<utility>` (for pair and move)
remove: `<functional>` once you've removed `std::function`?